From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: syphyr <syphyr@gmail.com>
Date: Wed, 3 Jan 2018 20:15:17 +0100
Subject: [PATCH] wifi: Add world regulatory domain country code

The World Regulatory Domain is the default country code used
for many wifi only devices. Adding this country code
provides a way to reselect this option if it is changed in
the wifi regional settings.

Also, adding this regional wifi option will fix the issue with
the wifi regional settings appearing to be "unselected" when
World Regulatory Domain is the default.

Change-Id: I2e519872954903575ac2fc47fa13d126bb5bac8e
---
 res/values/cm_arrays.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/res/values/cm_arrays.xml b/res/values/cm_arrays.xml
index 38568e4bafb..49e2a4f2d1e 100644
--- a/res/values/cm_arrays.xml
+++ b/res/values/cm_arrays.xml
@@ -269,6 +269,7 @@
 
     <!-- Wi-Fi settings. Presented as a list dialog to the user to choose the Wi-Fi region code. -->
     <string-array name="wifi_countrycode_entries">
+        <item>World Regulatory Domain</item>
         <item>United States</item>
         <item>Canada, Taiwan</item>
         <item>Germany</item>
@@ -284,6 +285,7 @@
     </string-array>
 
     <string-array name="wifi_countrycode_values" translatable="false">
+        <item>00</item>
         <item>US</item>
         <item>CA</item>
         <item>DE</item>
